lib/fluent/plugin/out_cloudwatch_logs.rb

```diff
@@ -12,6 +12,7 @@ class CloudwatchLogsOutput < BufferedOutput
 
     config_param :aws_key_id, :string, :default => nil, :secret => true
     config_param :aws_sec_key, :string, :default => nil, :secret => true
+    config_param :aws_credentials_retries, :integer, default: nil
```

Comment:
Would you change the name of this param to `aws_instance_profile_credentials_retries` ?